sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -1193,6 +1194,26 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -IsWBFluidEnabled
+
+Sets weather Whiteboard is enabled or disabled for OneDrive for Business users. Whiteboard on OneDrive for Business is automatically enabled for applicable Microsoft 365 tenants but can be disabled.
```

Comment:
```suggestion
Sets whether Whiteboard is enabled or disabled for OneDrive for Business users. Whiteboard on OneDrive for Business is automatically enabled for applicable Microsoft 365 tenants but can be disabled.
```

Reply:
@get-itips thank you for that correction. Can't believe I missed that.

Reply:
@FaganSC Can you apply the suggestion? cc @yogkumgit